src/apis/connecting/usePostCommentText.ts

```
@@ -0,0 +1,49 @@
+import { useMutation } from '@tanstack/react-query';
+import api from '../common/api';
+
+export interface PostCommentTextRequest {
+  courseId: number;
+  content: string;
+}
+
+export interface PostCommentTextResponse {}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)